ci/baremetal/baremetal-cluster.lokocfg.envsubst

```diff
@@ -8,8 +8,8 @@ cluster "bare-metal" {
   matchbox_client_key_path  = pathexpand("~/pxe-testbed/.matchbox/client.key")
   matchbox_endpoint         = "matchbox.example.com:8081"
   matchbox_http_endpoint    = "http://matchbox.example.com:8080"
-  cluster_name              = "mercury"
```

Comment:
I think this should require documentation changes, as role of `k8s_domain_name` field is now different. It's a breaking change.